src/server/saved_objects/client/lib/search_dsl/query_params.js

```diff
@@ -45,8 +45,9 @@ export function getQueryParams(mappings, type, search, searchFields) {
   if (search) {
     bool.must = [
       {
-        simple_query_string: {
+        query_string: {
           query: search,
+          minimum_should_match: search.split(' ').length,
```

Comment:
Is the purpose of this to make every term required? You could just change the `default_operator` option if that's the case.

Reply:
Ah, nice!